packages/react-core/src/components/Select/Select.tsx

```diff
@@ -62,14 +65,14 @@ export interface SelectProps extends MenuProps, OUIAProps {
   popperProps?: SelectPopperProps;
 }
 
-const SelectBase: React.FunctionComponent<SelectProps & OUIAProps> = ({
+const SelectBase: React.FunctionComponent<SelectBaseProps> = ({
   children,
   className,
   onSelect,
   isOpen,
   selected,
   toggle,
-  shouldFocusToggleOnSelect = false,
+  shouldFocusToggleOnSelect = true,
```

Comment:
Would be breaking.

Reply:
True. My thinking was that this had a potential of being included in the release of v5, but now it would be a problem for sure. This was a low impact and more of a suggested default behavior than anything.